src/main/java/org/opensearch/sdk/ExtensibleExtension.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.sdk;
+
+import java.util.List;
+
+/**
+ * An extension point for {@link Extension} implementations to be themselves extensible.
+ *
+ * This class provides a callback for extensible Extensions to be informed of other Extensions
+ * which extend them.
+ *
+ * @opensearch.api
```

Comment:
SDK doesn't use this annotation so it will fail compiling.  Please remove this.

We use package-based API definitions.